Storage/SQL/BrowserSchema.swift

```diff
@@ -148,15 +148,12 @@ open class BrowserSchema: Schema {
     public var version: Int { return BrowserSchema.DefaultVersion }
 
     let sqliteVersion: Int32
-    let supportsPartialIndices: Bool
 
     public init(logger: Logger = DefaultLogger.shared) {
-        let v = sqlite3_libversion_number()
-        self.sqliteVersion = v
-        self.supportsPartialIndices = v >= 3008000          // 3.8.0.
+        self.sqliteVersion = sqlite3_libversion_number()
         let ver = String(cString: sqlite3_libversion())
         self.logger = logger
-        logger.log("Init SQLite version: \(ver) (\(v)).",
+        logger.log("Init SQLite version: \(sqlite3_libversion()).",
```

Comment:
nit: Since you're not using `ver` in this log message it looks like line 154 can be removed. Also there's a subtle difference between the old log message and new, but it also looks like `ver` and `v` values might have been redundant.